source/Graphics.js

```diff
@@ -39,7 +39,7 @@ export default class Graphics
 		this._ViewTransform = mat2d.create();
 	}
 
-	initialize(cb, staticPath)
+	initialize(staticPath, cb)
```

Comment:
What was the reason for this change? Both Flare-React and our site implement the api with the cb first.

Reply:
Well our example uses it inverted, and since we always have callbacks as the last argument, I thought this was the case too.

Reply:
Ah ok, we should update the example or we'll risk breaking the other apps. Alternatively, we can make it invert the parameters as necessary internally...but probably easier to fix the example only for right now if it's the only thing that's broken.